<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>311</width>
      <height>194</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="btnOpenDir">
           <property name="text">
            <string>打开目录</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSelectPath">
           <property name="text">
            <string>选择保存路径</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnMergeImage">
           <property name="text">
            <string>合并图像</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>水平重叠</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editHOverlap"/>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>垂直重叠</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editVOverlap"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>航向线个数</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editHCount"/>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>俯仰线个数</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editVCount"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QCheckBox" name="cbShowLine">
           <property name="text">
            <string>显示原始图像边界</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbSaveColor">
           <property name="text">
            <string>输出24位彩色</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QRadioButton" name="rbForward">
           <property name="text">
            <string>从前往后覆盖</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbBackward">
           <property name="text">
            <string>从后往前覆盖</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbSymmetry">
           <property name="text">
            <string>对称覆盖</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbCenterReflect">
           <property name="text">
            <string>从中间向两侧</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
